src/assets/stylesheets/index.scss

```diff
@@ -109,6 +109,14 @@ a:focus-visible,
   transition: all ease-in-out 0.3s;
 }
 
+[data-scrivito-editing] {
+  a:focus-visible,
+  .btn:focus-visible,
+  :focus-visible {
+    outline: none !important;
+  }
+}
+
```

Comment:
* [ ] Rules in index.scss also affect the app and because of this WYSIWYG is broken in edit mode. Extension-only styles can be put into: `src/assets/stylesheets/scrivitoExtensions.scss`.